hamocc/mo_netcdf_bgcrw.F90

```diff
@@ -39,6 +39,7 @@ subroutine netcdf_def_vardb (kcid,kshort,yshort,kdims,kcdims,kcvarid,
     use netcdf,  only: nf90_double,nf90_noerr,nf90_put_att,nf90_def_var
     use mod_xc,  only: mnproc,xchalt
     use mod_dia, only: iotype
+    use mo_kind, only: rp
```

Comment:
In this routine/module the usage of DOUBLE is hard-coded a lot of places. Not sure how to handle this